pqcrypto-internals/build.rs

```diff
@@ -57,8 +57,12 @@ fn main() {
             .compile("keccak4x");
         println!("cargo:rustc-link-lib=keccak4x")
     } else if target_arch == "aarch64" && target_env != "msvc" {
+        let arch_flag = match target_os.as_str() {
+            "ios" => "-march=armv8-a+sha3",
+            _ => "-march=armv8-a",
```

Comment:
I don't think that this distinction is necessary, as we need to build with `+sha3` anyway (and it was likely just implied by my testing environment).